crates/trie/common/src/hashedstate.rs

```diff
@@ -292,6 +292,15 @@ impl HashedStorage {
     }
 }
 
+impl<'a, I> From<(AccountStatus, I)> for HashedStorage
+where
+    I: IntoIterator<Item = (&'a U256, &'a U256)>,
+{
+    fn from((status, storage): (AccountStatus, I)) -> Self {
+        Self::from_plain_storage(status, storage)
+    }
+}
```

Comment:
not a fan of this from conversion

I prefer having the from_plain_storage call visible

Reply:
right, me too, this is a hack in order to ensure `reth-optimism-consensus` stays no-std compat. otherwise I need to re-crate this issue which originally focused on moving `HashedStorage` to no-std compat crate https://github.com/paradigmxyz/reth/issues/14208

Reply:
nvm, issue wasn't hijacked, not sure why I thought this was in std crate first, my bad

Reply:
ah yes, because I used the trie test utils first to calculate the storage root